docs/structured-streaming-kafka-integration.md

```diff
@@ -379,7 +379,25 @@ The following configurations are optional:
   <td>string</td>
   <td>spark-kafka-source</td>
   <td>streaming and batch</td>
-  <td>Prefix of consumer group identifiers (`group.id`) that are generated by structured streaming queries</td>
+  <td>Prefix of consumer group identifiers (`group.id`) that are generated by structured streaming
+  queries. If "kafka.group.id" is set, this option will be ignored. </td>
```

Comment:
nit: Given that other option is wrapped with `, might better to follow same rule for consistency.

Reply:
> nit: Given that other option is wrapped with `, might better to follow same rule for consistency.

We don't have such rule. See the doc of `failOnDataLoss`

Reply:
Yup. I think I chose word incorrectly. Many options are wrapped with \` so felt we are having implicit rule on that. please ignore if the approach on representation is already not consistent.